### ISSUE TITLE
Integer literal bug

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/ergo#484]
The JavaScript code generation for integer literals is wrong.

This can be seen on the following unit test. 
[integertest.zip](https://github.com/accordproject/ergo/files/2677077/integertest.zip)

To test:
```
bash-3.2$ ergorun ./model.cto ./logic.ergo --contractname org.accordproject.integertest.IntegerTest --contract ./contract.json --request ./request.json --state ./state.json 
12:34:47 - info:
{
  "response": {
    "test1": {
      "nat": 0
    },
    "test2": 0,
    "test3": 0,
    "$class": "org.accordproject.integertest.Response"
  },
  "state": {
    "$class": "org.accordproject.cicero.contract.AccordContractState",
    "stateId": "1"
  },
  "emit": []
}
```

Note the:
```
    "test1": {
      "nat": 0
    },
```
has an extraneous `nat` field.

**Comment by jeromesimeon** [comment]
The issue can be seen on the generated JavaScript code:
```
bash-3.2$ ergoc ./model.cto ./logic.ergo 
Compiling Ergo './logic.ergo' -- creating './logic.js'
```

line 43 of the compiled `logic.js`:
```
      res2 = {"left" : concat(concat({"response": brand(["org.accordproject.integertest.Response"],concat(concat({"test1": {"nat": 0.0}}, {"test2": natPlus({"nat": 0.0}, {"nat": 0.0})}), {"test3": Math.trunc(vp1)}))}, {"state": vlstate}), {"emit": vlemit})};
```

shows the extraneous `nat` fields which are inconsistent with the serialization expected by JSON data validated with Concerto.

**Comment by jeromesimeon** [comment]
Fixed in PR #486
